yjit/src/cruby.rs

```diff
@@ -290,7 +290,7 @@ pub struct rb_iseq_t {
         core::marker::PhantomData<(*mut u8, core::marker::PhantomPinned)>,
 }
 
-#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+#[derive(Copy, Clone, PartialEq, Eq, Debug, Hash)]
```

Comment:
It's not safe to use any `VALUE` as a key in a hash map because of values movable by the Ruby GC.